bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -402,9 +402,9 @@
         }]
       end
 
-      it "doesn't just fall back to latest_version" do
+      it "unlocks the latest version" do
```

Comment:
Don't really understand the wording on the existing test, from my understanding and digging into to the version resolver, we'll try to unlock the latest version regardless of what's in the manifest currently because we pass `unlock_requirement = true`